src/components/_btn.css

```diff
@@ -59,7 +59,20 @@ Use the button classes on and `<a>`, `<button>`, `<input>` elements.
 .btn-hover-red-text { text: #ef3934; }
 
 .btn-min-width {
-  min-width: var(--space-7);
+  min-width: 92px;
+}
+
+.btn-simple {
+  border: unset;
+  background-color: inherit;
+  border-radius: 4px;
```

Comment:
Is this supposed to be smaller than the normal button-radius?

Reply:
The design feedback was the feature UI uses 4px, not 6px

Reply:
👍